Umbraco-Cloud/Set-Up/Dedicated/index.md

```diff
@@ -0,0 +1,53 @@
+---
+versionFrom: 8.0.0
+versionTo: 10.0.0
+---
+
+# Move your Umbraco Cloud project to dedicated resources
+
```

Comment:
How about we add this sentence here at the top, in order to make it clear what the terms mean, and give an overview of the feature?

```suggestion

*Dedicated Resources* is a feature on Umbraco Cloud that gives you the option to move your project to a dedicated server. You can choose between a number of *dedicated plans* depending on the amount of resources you will need for your project.

```